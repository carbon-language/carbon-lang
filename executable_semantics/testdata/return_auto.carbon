// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
// RUN: ../executable_semantics %s | \
// RUN:   FileCheck %s

package ExecutableSemanticsTest api;

fn F() -> auto {
  return 0;
}

fn main() -> i32 {
  return F();
}

// AUTOUPDATE: ../executable_semantics %s
// CHECK: COMPILATION ERROR: executable_semantics/testdata/return_auto.carbon:10: syntax error, unexpected AUTO
