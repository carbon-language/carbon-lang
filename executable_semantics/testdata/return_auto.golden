COMPILATION ERROR: 5: syntax error, unexpected AUTO
EXIT CODE: 255
